test/render/example-test-cases.js

```diff
@@ -0,0 +1,39 @@
+import {setPathPrefix} from '@luma.gl/core';
+
+const RESOURCE_PATH = 'https://raw.githubusercontent.com/uber/luma.gl/master';
+
+const examples = {
+  'lesson-01': require('../../examples/lessons/01/app').default,
+  'lesson-02': require('../../examples/lessons/02/app').default,
+  'lesson-03': require('../../examples/lessons/03/app').default,
+  'lesson-04': require('../../examples/lessons/04/app').default
+  // 'lesson-05': require('../../examples/lessons/05/app').default,
```

Comment:
@ibgreen The examples that use textures cannot be tested with the current API design. When using the new async Texture constructor, we are unable to detect when textures are loaded and rendered.

Reply:
Yes, I thought about e.g. storing a load promise on the textures. It would only  cover the case of initially setting the texture to url.

But if multiple textures are used, it could get a bit pedantic.

Other options
* have `draw` return a value that indicates if it was able to draw
* make the program `isRenderable` method public
* Add a `Program.getRenderablePromise()` that returns a composite promise of all texture valued uniforms' loading promises.

Whatever we do, ultimately there are going to be examples with other loading issues that need to be captured, like async props in deck.gl.